### ISSUE TITLE
Game is blurry when using 4x SuperSampling.

### ISSUE THREAD

**Issue by MrBox15** [CJ-SPT/PerformanceImprovements#14]
​Hey, just to let you know I "might" found an issue or an incompatibility with your mod. It seems like the Optic downscalling is alway enabled even if I am not aiming and the downscaling when ads in the mod option is disabled.
​
​I play in 4x Supersampling and it look like I have 0.5x downscalling. When I put to 1x no supersampling, it look way better.
​
​I tried removing a couple mod and the one causing this was yours.

The Optic Downscalling option was disabled and also the Experimental Graphic Option.
​
​This problem appeared somewhere in the last 3 to 4 week cause I took a pause of playing SPT during this period and before it was looking crisp and using Performance Improvements.

I updated the mod and now I have this problem. So I dunno if its intended or a bug or incompatibility. 

If you need logs or anything else let me know

**Comment by arheor** [comment]
Yes, I have a similar problem. If I enable TAA ingame and 0.5x in the Performance Improvement settings, then my game becomes blurry after aiming and does not return to its original state. I use a translator, ask if something is not clear
